client/src/wh/components/forms/views.cljs

```diff
@@ -152,24 +152,28 @@
   [value options]
   (let [dirty (reagent/atom false)
         focused (reagent/atom false)]
-    (fn [value {:keys [suggestions dirty? error force-error? read-only on-select-suggestion hide-icon?] :as options}]
+    (fn [value {:keys [suggestions dirty? error force-error? read-only on-select-suggestion on-remove hide-icon?] :as options}]
       (when (and (not (nil? dirty?))
                  (boolean? dirty?))
         (reset! dirty dirty?))
       (let [value (or value (:value options))
             suggestable? (and (or (seq suggestions) on-select-suggestion) (not hide-icon?))
-            show-suggestion? (and (seq suggestions) @focused (not read-only))]
+            show-suggestion? (and (seq suggestions) @focused (not read-only))
+            removable? (and value (not @focused))]
```

Comment:
`removable?` should also only be true if `on-remove` is a valid seq. At the moment this is causing the remove icons to appear on other text fields:
![Screenshot_20190930_105029](https://user-images.githubusercontent.com/812199/65868673-a1171e80-e370-11e9-9b9d-62b03eaa9b4e.png)

Reply:
Thank you for the review and sorry for missing this